docs/cue/reference/remap/functions/assert_eq.cue

```diff
@@ -0,0 +1,60 @@
+package metadata
+
+remap: functions: assert_eq: {
+	category: "Debug"
+
+	description: """
+		Asserts that two expressions, `left` and `right`, have the same value. If the values are
+		equivalent, `true` is returned, otherwise `false` is returned. You can specify an optional
+		`message` to append to the error output.
```

Comment:
I think this should more closely match the description in `assert` as it is essentially the same as `assert(left == right)`. Something like:

```suggestion
Asserts that two expressions, `left` and `right`, have the same value. The program is aborted with the `message` if they are unequal.
```

And then have a similar `notice`:

```
		The `assert_eq` function should be used in a standalone fashion and only when you want to abort the program. You should avoid it in logical expressions and other situations in which you want the program to continue if the condition evaluates to `false`.
```